### PR TITLE
Create a new React app called 'hello-react-front-end'

### PR DESCRIPTION
**Pull request by diegoh40**
- Initialize your project with Git.
- Create the App component with react-router.
- Create the Greeting component that will display a greeting. Set it up as a route in your App component.
- Create a store, an action and a reducer that will connect to you API endpont to get the random greeting.
- Display the random greeting in your Greeting component.

this is the link of Backend PR  https://github.com/diegoh40/hello-rails-react-backend/pull/1

### CODE REVIEW

File: src/redux/message/message.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+const GET_MESSAGE_API = 'messageReducer/GET_MESSAGE_API';
+
+const initialState = [];
+
+// const getMessage = (payload) => ();
```
Comment (Davidosky007) [inline src/redux/message/message.js]: * Kindly remove all commented code from your codebase as it makes your code look bulky and unprofessional 🙏

File: src/redux/message/message.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  switch (action.type) {
+    case GET_MESSAGE_API:
+      return action.payload;
+    default:
+      return state;
+  }
+};
+
+export default messageReducer;
+// export { getMessageAPI };
```
Comment (Davidosky007) [inline src/redux/message/message.js]: * Kindly remove all commented code from your codebase as it makes your code look bulky and unprofessional 🙏

File: src/redux/message/message.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+const initialState = [];
+
+// const getMessage = (payload) => ();
+
+export const getMessageAPI = () => async (dispatch) => {
+  const res = await fetch('http://localhost:3000/greetings/show', {
+    mode: 'no-cors',
+  });
+  const data = await res.json();
```
Comment (Davidosky007) [inline src/redux/message/message.js:7]: * I get this error in the console while I run the frontend of your app, Kindly fix this as no random greeting is being rendered 
image

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
Hi @diegoh40 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
## Highlights
* No linter errors 👍 
* All necessary links provided 👍 
* Well written PR description 👍 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @davidsoky007 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/redux/message/message.js (hunk 3 above)
Comment (diegoh40) [reply src/redux/message/message.js:7]: the error was corrected and updated, I add the screenshot of my browser:

image

File: src/redux/message/message.js (hunk 2 above)
Comment (diegoh40) [reply src/redux/message/message.js]: done

File: src/redux/message/message.js (hunk 1 above)
Comment (diegoh40) [reply src/redux/message/message.js]: done

**Review by m05e5 — APPROVED** [review]
Hi @diegoh40 

You did great. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
CleverScornfulBuffalo-size_restricted

## Highlights
✔️ Descriptive PR 
✔️ your linters are all ok

## Optional suggestions


_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_**m05e5**_

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**